drivers/lora/sx12xx_common.c

```diff
@@ -33,6 +33,7 @@ static struct sx12xx_data {
 	const struct device *dev;
 	struct k_poll_signal *operation_done;
 	lora_recv_cb async_rx_cb;
+	void *user_data;
```

Comment:
Please rename the device variables to `async_user_data` so it is clearer that the user data is related to `async_rx_cb`.

Reply:
Done